src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -492,6 +496,7 @@ export default function LoadedRow({
   // TODO: currency logo sizing mobile (32px) vs. desktop (24px)
   return (
     <StyledLink
+      style={style}
```

Comment:
same comment re: spreading element styles instead of enumerating each pass-through prop

Reply:
responsed in TokenTable comment